mon-pix/tests/acceptance/authentication/signup-test.js

```diff
@@ -45,6 +45,10 @@ module('Acceptance | authentication | Signup', function (hooks) {
     assert.dom(signupHeading).exists();
     const loginButton = screen.queryByRole('link', { name: t('pages.sign-up.actions.login') });
     assert.dom(loginButton).exists();
+    const otherAuthenticationProvidersTitle = screen.queryByText(
+      t('components.authentication.other-authentication-providers.signup.heading'),
+    );
+    assert.dom(otherAuthenticationProvidersTitle).exists();
```

Comment:
On peut utiliser ok pour éviter le exists
```suggestion
    assert.ok(otherAuthenticationProvidersTitle);
```